src/main/java/org/codehaus/mojo/wagon/CopyMojo.java

```diff
@@ -60,6 +61,20 @@ public class CopyMojo
     @Parameter( property = "wagon.caseSensitive")
     private boolean caseSensitive = true;
 
+    /**
+     * Local directory to store downloaded artifacts.
+     */
+    @Parameter( property = "wagon.downloadDirectory")
+    private File downloadDirectory;
+
+
+    /**
+     * Download files that doesn't exist in local directory.
+     * Upload files that are not yet in target.
+     */
+    @Parameter( property = "wagon.incremental")
+    private boolean incremental;
```

Comment:
I fear that using a `boolean` will hit us later. Maybe there are other ways to use this parameter in the future, e.g. pick up partial downloads. This should be something like an Enum: 

```
enum ContinuationType {
  /** old default */
  NONE,
  /** skip existing files */
  ONLY_MISSING,
  /** continue partials */
  CONTINUE_PARTIALS;
}
```

Just an idea, though.

Reply:
done